server/deskroom/config.py

```diff
@@ -24,8 +24,9 @@ class Settings(BaseSettings):
     VERSION: str = Field(__version__, alias="VERSION")
 
     # supabase
-    SUPABASE_URL: str = ""
-    SUPABASE_ANON_KEY: str = ""
+    SUPABASE_URL: str = Field("", alias="DSKRM_SUPABASE_URL")
+    SUPABASE_ANON_KEY: str = Field("", alias="DSKRM_SUPABASE_ANON_KEY")
+    SUPABASE_SERVICE_ROLE_KEY: str = Field("", alias="DSKRM_SUPABASE_SERVICE_ROLE_KEY")
```

Comment:
```suggestion
    SUPABASE_URL: str = Field("")
    SUPABASE_ANON_KEY: str = Field("")
    SUPABASE_SERVICE_ROLE_KEY: str = Field("")
```

pydantic 2.0 이상, pydantic_settings 모듈을 사용하면 model_config 의 env_prefix가 자동으로 붙어서 alias를 지정할 필요가 없어졌습니다. 다른 field들 중 alias가 있는 것들은 외부 디펜던시 (env 이름이 이미 지정이 되어있는 경우) 가 있는 것들입니다. 
(사실 Azure 쪽은 아닌데 어쩌다보니 그냥 넘어간듯)

Reply:
처음에 `alias` 값으로 `SUPABASE_URL` 만 하니, 인식이 안되었습니다. 위처럼 `DSKRM_SUPABASE_URL` 를 하니 인식이 되더군요. 제가 뭔가 놓치고 있는게 있을까요?

Reply:
넵 놓치고 있는 게 있는 거 같아요 
https://docs.pydantic.dev/latest/concepts/pydantic_settings/
여기 한번 읽어보심 이해가 갈듯. 당연히 alias 입력을 안하면 SUPABASE_URL 로 인식이 안됩니다.